src/Sulu/Bundle/ContentBundle/Document/BasePageDocument.php

```diff
@@ -157,6 +157,8 @@ class BasePageDocument implements
      */
     protected $locale;
 
+    protected $originalLocale;
```

Comment:
This variable has been introduced in this PR, but it's not used anywhere. Do you really need this?